on-prem-to-cloud/README.md

```diff
@@ -0,0 +1,61 @@
+# Connectivity: Accessing a Workload in SAP BTP, Kyma Runtime From the Corporate Network
+
+## Context
+
+If you are using a corporate network, you can access a workload located in SAP BTP, Kyma runtime. The access is possible using the SAP BTP, Connectivity. See the following diagram that presents all the involved elements and the high-level flow:
```

Comment:
better to rephrase. If you have the requirement to access the worklaods deployed on kyma without exposing them over the internet, you can use the Connectivity service channel fetaure along with Kyma. Add the sap help link for service channel